src/blockMonitor/blockProcessor.ts

```diff
@@ -127,7 +128,7 @@ export class BlockProcessor<TBlock extends IBlockStub> extends StartStopService
 
     private mBlockCache: BlockCache<TBlock>;
 
-    private newHeadListeners: NewHeadListener<TBlock>[] = [];
+    public NewHead = new BlockEvent<TBlock>();
```

Comment:
At the moment I think we're only using capitalised names for static properties

Reply:
I know it's not our convention, but for some reason it felt more natural.
Anyway, changed to lowercase names in 749e18e.